src/Host/Broker/Impl/Interpreters/InterpreterManager.cs

```diff
@@ -40,17 +40,17 @@ public class InterpreterManager {
 
         private IEnumerable<Interpreter> GetInterpreters() {
             if (_options.AutoDetect) {
-                _logger.LogTrace("Auto-detecting R ...");
+                _logger.LogTrace(Resources.Trace_AutoDetectingR);
 
                 var engines = new RInstallation().GetCompatibleEngines();
                 if (engines.Any()) {
                     foreach (var e in engines) {
                         var detected = new Interpreter(this, Guid.NewGuid().ToString(), e.Name, e.InstallPath, e.BinPath, e.Version);
-                        _logger.LogTrace($"R {detected.Version} detected at \"{detected.Path}\".");
+                        _logger.LogTrace(string.Format(Resources.Trace_DetectedR, detected.Version, detected.Path));
```

Comment:
You don't need to use `string.Format` - they provide `params` overloads for `Log...` that do that for you. They have one other advantage, in that they also retain the format arguments as event data, so an advanced logger can log the values separately from the formatted message, for latter filtering on the values.